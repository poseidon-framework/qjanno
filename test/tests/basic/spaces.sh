qjanno "SELECT * FROM data/spaces.csv"
