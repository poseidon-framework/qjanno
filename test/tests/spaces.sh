qhs -H -O "SELECT * FROM spaces.csv"
