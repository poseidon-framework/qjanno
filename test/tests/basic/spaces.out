.-----------------.-----.-----.-----.
|   source_file   | foo | bar | baz |
:=================:=====:=====:=====:
| data/spaces.csv | a0  | 1   | a2  |
| data/spaces.csv | b0  | 3   | b2  |
| data/spaces.csv | c0  | c2  |     |
'-----------------'-----'-----'-----'
